app/components/inspector/InstructionsSection.tsx

```diff
@@ -37,32 +40,32 @@ function InspectorInstructionCard({
     const programName = getProgramName(programId.toBase58(), cluster);
     const anchorProgram = useAnchorProgram(programId.toString(), url);
 
-    const transactionInstruction = intoTransactionInstructionFromVersionedMessage(ix, message, programId);
+    const transactionInstruction = intoTransactionInstructionFromVersionedMessage(ix, message);
 
     if (anchorProgram.program) {
+        const programName = getAnchorProgramName(anchorProgram.program) ?? 'Unknown Program';
```

Comment:
Please remove this. This call can throw, and it would not get caught in the ErrorBoundary.

Also if we fail to deserialize the Anchor instruction, we may not have an Anchor program name

Reply:
Fixed